database/seed.js

```diff
@@ -0,0 +1,82 @@
+const faker = require('faker');
+const db = require('./index.js');
+const Promise = require('bluebird');
+
+const seedRewards = () => {
+  const rewards = [];
+  let tiers;
+  for (let i = 1; i < 101; i++) {
+
+    if (i % 10 === 0 && i % 8 !== 0) {
+      tiers = [1, 5, 10, 25, 50, 75, 100, 250, 500, 750];
+    } else if (i % 8 === 0) {
+      tiers = [1, 5, 10, 25, 50];
+    } else {
+      tiers = [1, 5, 10, 25, 50, 75, 100];
+    }
+
+    for (let j = 0; j < tiers.length; j++) {
+      let isLimited = false;
+      let limitCount = null;
+      let estDeliv = faker.date.month() + '' + faker.random.number({min: 2019, max: 2022});
+      let backers = faker.random.number(500);
+
+      if (j === 8) {
+        isLimited = true;
+        limitCount = 30;
+        backers = faker.random.number(30);
+      }
+      if (j === 9) {
+        isLimited = true;
+        limitCount = 10;
+        backers = faker.random.number(10);
+      }
+
+      rewards.push({
+        projectId: i,
+        pledgeAmount: tiers[j],
+        name: faker.lorem.words(),
+        description: faker.lorem.paragraph(),
+        item1: faker.lorem.words(),
+        item2: faker.lorem.words(),
+        item3: faker.lorem.words(),
+        isLimited: isLimited,
+        limitCount: limitCount,
+        estDeliv: estDeliv,
+        shipsTo: faker.lorem.words(),
+        backers: backers
+      });
+    }
+  }
+
+  let rewardPromises = rewards.map((reward) => {
+    return db.Reward.create(reward)
+      .catch((err) => {
+        return Promise.resolve();
+      });
+  });
+
+  return Promise.all(rewardPromises);
+
```

Comment:
Is there a batchCreate or bulkCreate method that might let you take out the promises?

Reply:
@jordanstholmes I think there might be a bulkCreate method but I'm not sure. I'll have to double-check the Sequelize docs. I'll probably leave part as is for the time being and refactor to use bulkCreate at a later point if I have some extra time. Thanks so much for the feedback!

Reply:
Fo sho, bro